packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -1,58 +1,106 @@
 <template>
```

Comment:
This file should probably be named differently, since it's not just a dialog. It could be a "dialog" (modal) or an alert, right?

Reply:
I might be inclined to have two separate components. One for dialogs, one for alerts, and we should one or the other only. Not sure, though.

Reply:
I've removed alert from the component